build/prepare/CentosCoreclrAndCoreFxDependencies.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+	<CentosCoreclrAndCoreFxDependencies Include="unzip" />
```

Comment:
Because it isn't VS. 

![image](https://cloud.githubusercontent.com/assets/8291187/17000599/d66bc20a-4e88-11e6-8755-587c34338af1.png)